pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -129,3 +131,150 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 	}
 	return nil
 }
+
+func CheckContainerReferences(cluster cluster.Cluster, tenantId string, containerConfig *dockerclient.ContainerConfig) error {
+	log.Debugf("CheckContainerReferences containerConfig: %+v", containerConfig)
+	// create arrays of container references to pass to getIDsFromContainerReferences
+	// create volumesFrom array
+	volumesFrom := containerConfig.HostConfig.VolumesFrom
+
+	// create affinity array
+	affinityRef := make([]string, 0)
+	env := containerConfig.Env
+	var affinityIndex int
+	affinityLabelFirst := false
+	for i, e := range env {
```

Comment:
I think It makes perfectly sense to have multiple different  labels for affinity and even have anti affinity together with them ... I think the whole affinity should be reviewed.   I suggest to remove it from this PR... In any case this was kind of workaround for bad written compose files for swarm - instead of fixing the compose file (using links, network, etc..) we force all containers to the same node ...